connectors/CommerceCatalogServiceGraph.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "CommerceCatalogServiceGraph",
+  "version": "1.0.0",
+  "author": "lradu",
+  "description": "Adobe Commerce Catalog Service",
+  "provider": {
+    "name": "CommerceCatalogServiceGraph",
+    "handler": {
+      "graphql": {
+        "endpoint": "https://catalog-service.adobe.io/graphql/",
```

Comment:
@LucianRadu  this looks to me as an external endpoint for catalog services. Based on our conversation with Sandra, Misha and Fahd I was expecting internal endpoints.